platforms/corda-ent/configuration/roles/setup/node_registration/tasks/main.yaml

```diff
@@ -51,7 +51,7 @@
     charts_dir: "{{ org.gitops.chart_source }}"
     component_name: "{{ peer.name | lower }}registration"
     component_ns: "{{ org.name | lower }}-ent"
-    node_name: "{{ peer.name | lower }}registration"
+    node_name: "{{ peer.name | lower }}"
```

Comment:
Why is this changed? This was a fix so that the release names are different for node vs node-registration

Reply:
this is changed because the word "registration" was coming twice...e.g. "manufacturerregistration-registration", as the app_name was taking {{ node_name | lower }}-registration...to remove the redundant occurrence of the word "registration" this change is done.